src/olympia/access/acl.py

```diff
@@ -40,7 +38,8 @@ def action_allowed_user(user, app, action):
         return False
 
     return any(
-        match_rules(group.rules, app, action) for group in user.groups_list)
+        match_rules(group.rules, permission.app, permission.action)
+        for group in user.groups_list)
```

Comment:
I actually wonder if we should enforce that a permission **has to be** in `constants.permissions` to make sure we don't allow any none-constants?